### DISCUSSION TITLE
qt6ct

### DISCUSSION THREAD

**Discussion by shadow-absorber** [catppuccin/catppuccin#2624] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

as shown in the pull request for qt5ct here: [https://github.com/catppuccin/qt5ct/pull/5](https://github.com/catppuccin/qt5ct/pull/5) qt6ct is nearly identical just having one extra field. opening this to discuss if that should be in its own repo or merged into a qtct repo as one

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
I wouldn't want to create a separate repository with nearly identical files, closing this discussion in favour of tracking progress in https://github.com/catppuccin/qt5ct/issues/8
